examples/http3_client.py

```diff
@@ -264,7 +264,7 @@ async def perform_http_request(
         )
         method = "POST"
     else:
-        http_events = await client.get(url)
+        http_events = await client.get(url, headers={'pragma': 'azion-debug-cache'})
```

Comment:
How about adding a parameter for the caller to provide the additional headers ?

Reply:
I prefer to avoid add azion specific behavior in an external library.

Reply:
> I prefer to avoid add azion specific behavior in an external library.

1. This is no longer an external library. Check the github organization that is hosting this repository. Do we have any plans to merge this repository into the original opensource repository?
2. Please refer to "E2E Tests Development Guidelines" document, Section Good Practices for Reviewers, item 3: `Avoid requesting changes due to personal preferences. Consider the following questions: Is this somehow gonna break one or more tests? Is this a really, really a bad practice that MUST BE avoided? If so, please explain it.` - If the sentence starts with `I prefer ...` then it's already not OK. Is there any justification for that?

> How about adding a parameter for the caller to provide the additional headers ?

Can do that, but remember: time is not on our side regarding http3

Reply:
OK, changing the approach of my explanation. This step will fails without the headers as parameters:
https://github.com/aziontech/qa-e2e-tests/blob/27b3ae56ede0ed903bf709ac31b1c2c5954d963a/features/steps/api/cache/cache_control.py#L84

Reply:
Not really sure what is going on ...
But this branch is not getting pulled to my local machine. So closing this PR, and opened a new one:
https://github.com/aiortc/aioquic/pull/362